src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/blackrock_depths.cpp

```diff
@@ -124,7 +124,6 @@ enum GrimstoneTexts
     SAY_TEXT6          = 5
 };
 
-/// @todo implement quest part of event (different end boss)
```

Comment:
I am fairly certain that this needs to stay. This line is about the Ring of the Law (NPC Grimstone).
Your change in `instance_blackrock_depths.cpp` is about Princess Moira, not a random elite mob.
See also https://wow.gamepedia.com/High_Priestess_of_Thaurissan